### ISSUE TITLE
Package bots running dart2js on jsshell fail in test:package

### ISSUE THREAD

**Issue by whesse** [dart-lang/package-bots#10]
Bots like package-linux-collection try to run all the tests with jsshell, and this fails because jsshell does not support non-zero timers, and package:test uses them for its Invoker.heartbeat.

We should stop running package tests on jsshell, in my opinion, since all of them should be using package:test.

@nex3 @jakemac53

**Comment by nex3** [comment]
Which bots are still trying to run on jsshell?

**Comment by rakudrama** [comment]
Under which heading is package-linux-collection?

If we are testing on firefox and the bot(s) are healthy then I think removing jsshell is OK. Otherwise we need coverage of jsshell or firefox.

Is this something that could be fixed by porting timer code from [preambles/d8.js](https://github.com/dart-lang/sdk/blob/0c1e85ebadaa0680d4ff30acc1f3ac823c53756e/sdk/lib/_internal/js_runtime/lib/preambles/d8.js) to [preambles/jsshell.js](https://github.com/dart-lang/sdk/blob/0c1e85ebadaa0680d4ff30acc1f3ac823c53756e/sdk/lib/_internal/js_runtime/lib/preambles/jsshell.js)?

**Comment by whesse** [comment]
I don't think it can be fixed, because there are just no non-zero timers available in jsshell.  You would need to implement them with DateTime and polling, I think.  I will remove the jsshell testing.

The bots running them are those without a .test_config file in the top-level directory of their package.
